satellite.py

```diff
@@ -40,6 +40,14 @@
 import neal
 import sys
 from dwave.system import LeapHybridSampler
+# Pr(at least one event)=1−Pr(none of the events)
+# https://math.stackexchange.com/questions/85849/calculating-the-probability-that-at-least-one-of-a-series-of-events-will-happen
+def calculateScore(constellation, data):
```

Comment:
Another style nitpick, but can you please rename the function to `calculate_score` to match our usual style?
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
done